readthedocs/core/utils/__init__.py

```diff
@@ -207,17 +214,19 @@ def trigger_build(project, version=None, commit=None, record=True, force=False):
     :rtype: tuple
     """
     log.info(
-        'Triggering build. project=%s version=%s commit=%s',
+        'Triggering build. project=%s version=%s commit=%s build=%s',
         project.slug,
         version.slug if version else None,
         commit,
+        build.pk
```

Comment:
In the build list page `build_pk` is not passed in the form. So I think trying to get `build.pk` for logging will cause an error.
```suggestion
        build.pk if build else None
```